numpy/random/_generator.pyx

```diff
@@ -559,39 +558,39 @@ cdef class Generator:
             high = low
             low = 0
 
-        dt = np.dtype(dtype)
-        key = dt.name
-        if key not in _integers_types:
```

Comment:
Is this variable used any more, or can it be removed?

Reply:
Do you mean __integers_types_? removed.